### PR TITLE
Misc changes

### PR DESCRIPTION
**Pull request by symroe**
See each commit message + let's chat about this

### CODE REVIEW

**Comment by chris48s** [comment]
Can you chuck the issue numbers for any relevant issues in here

**Comment by symroe** [comment]
e40a511 and 560da95 don't have issues AFIK.  
db0adb9 closes #106, as per the commit message.

File: every_election/apps/elections/views/id_creator.py [hunk 1]
```diff
@@ -195,9 +195,12 @@ def get_context_data(self, form, **kwargs):
             all_data['radar_id'] = self.storage.extra_data.get('radar_id', None)
 
         context['all_data'] = all_data
-        all_ids = create_ids_for_each_ballot_paper(all_data, self.get_election_subtypes())
-        # all_ids = create_ids_grouped(all_data, self.get_election_subtypes())
-        context['all_ids'] = all_ids
+        if kwargs['step'] in ["review", "done"]:
```
Comment (chris48s) [inline every_election/apps/elections/views/id_creator.py]: I tried running this locally and went through the wizard to create an ID for the Alyn & Deeside by-election on 6th Feb. This line is throwing `KeyError: 'step'` at the last phase (i.e: after the review when I go to actually create the election object). Can you have a look please.

Comment (symroe) [reply every_election/apps/elections/views/id_creator.py]: Fixup pushed

**Comment by chris48s** [comment]
As noted yesterday, if you're re-running the register imports on prod to deploy this, you may also be able to close off some/all of #16, #69 and #94 so worth reviewing those issues when you deploy :)

**Pull request merged**